src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/ui-data/lambda-metrics.mdx

```diff
@@ -0,0 +1,385 @@
+---
+title: Available Lamda metrics
+metaDescription: Available Lambda metrics in New Relic.
+freshnessValidatedDate: never
```

Comment:
@cbehera-newrelic you could add a freshness date even thought this is a new doc.